scss/_patterns_list-tree.scss

```diff
@@ -46,26 +42,32 @@
     @include vf-focus;
 
     margin-top: $sp-xx-small;
-    padding-left: 0.8rem;
+    padding-left: map-get($grid-margin-widths, small);
+
+    @media (min-width: $threshold-4-6-col) {
```

Comment:
In side navigation this may have something to do with grid alignment (the grid margin is responsive - smaller on mobile), and the indentation of the side nav may align with this (or at least this may have been intention).

Does it need to apply to all nesting levels (or only top layer that may need alignment with the grid) I don't know.

I guess for the sake of updating this component we can attempt to implement simpler version first, and either wait for design review from Lyubo, or update it later when he can give some design input on it.

Reply:
Removed the responsive layout changes